db/migrations/25-12-06-add-export-request.sql

```diff
@@ -0,0 +1,36 @@
+create type export_request_status as enum ('PENDING', 'IN_PROGRESS', 'COMPLETE', 'FAILED');
+
+create table export_request (
```

Comment:
I'd prefer to track all of this in the `payload` and `data` columns of the `job` table. Both of those columns can hold arbitrary JSON. Jobs are ephemeral so I find referential integrity of job data to be less important and having a single mechanism for tracking job progress is useful. Do you have any concerns with this approach?